_layouts/front.html

```diff
@@ -98,18 +98,49 @@ <h3>Mentorship</h3>
                 <h2 class="section-heading">Get Involved!</h2>
                 <hr class="dark">
                 <p class="text-muted">
-                    We are always looking for high school and college students to help us support our community! 
-                    Whether you are good at art, like to post on social media, have great scores on tests and in school, 
-                    want to share your college application journey with others, or like writing articles, 
+                    We are always looking for high school and college students to help us support our community!
+                    Whether you are good at art, like to post on social media, have great scores on tests and in school,
+                    want to share your college application journey with others, or like writing articles,
                     there's always a position for you on our team!</p>
                 <a href="/get-involved" class="btn btn-primary btn-xl">Learn More!</a>
             </div>
         </div>
     </div>
   </section>
 
+  <section class="bg-light" id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-3">
+          <h3 class="section-heading">Social Media</h3>
+          <ul class="list-unstyled" id="footer-links">
+            <li><a href="https://discord.myeduvisa.org">Discord</a></li>
+            <li><a href="https://www.instagram.com/myeduvisa/">Instagram</a></li>
+            <li><a href="https://www.facebook.com/teameduvisa/">Facebook</a></li>
+            <li><a href="https://www.youtube.com/channel/UC6KGOBlBVm8yp172XAoNKdQ">YouTube</a></li>
+            <li><a href="https://www.linkedin.com/company/myeduvisa/">LinkedIn</a></li>
+            <li><a href="https://twitter.com/myeduvisa">Twitter</a></li>
+            <li><a href="https://www.reddit.com/user/eduvisa/">Reddit</a></li>
+            <li><a href="https://linktr.ee/eduvisa">Linktree</a></li>
+          </ul>
+        </div>
+
+        <div class="col-sm-3">
+          <h3 class="section-heading">Join Us</h3>
+          <ul class="list-unstyled" id="footer-links">
+            <li><a href="/get-involved"> Get Involved </a></li>
+            <li><a href="/get-involved/tutoring"> Tutoring </a></li>
+            <li><a href="/get-involved/mentorship"> Mentorship </a></li>
+            <li><a href="/get-involved/specializations"> Specializations </a></li>
+            <li><a href="mailto:team@myeduvisa.org">Contact Us</a></li>
+            <li><a href="https://donate.myeduvisa.org">Donate</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </section>
+
```

Comment:
I think it'd be good to have a column for our programs too, something like: 

**Our Community Programs**:
* [Apply for a Tutor](getatutor.myeduvisa.org)
* [Apply for a Mentor](getamentor.myeduvisa.org)
* [Check Out Our Upcoming Events](lu.ma/eduvisa)

Reply:
Uhh apparently I don't know how to link these properly on Github:

The links are:
* getatutor.myeduvisa.org
* getamentor.myeduvisa.org
* lu.ma/eduvisa

If #5 goes through first, the links should be updated to:
* myeduvisa.org/programs/tutoring (should be linked using `href=/programs/tutoring`)
* myeduvisa.org/programs/mentorship (should be linked using `href=/programs/mentorship`)
* lu.ma/myeduvisa

If this PR goes through before #5, I'll take care of updating the footer links

Reply:
Oooo yeah good idea. We really need that event calendar